Automattic/sensei
# Remove category question feature flag

Closes #4057 

### Changes proposed in this Pull Request

* If we're happy with #4080 after merging, this removes the feature flag.

### Testing instructions

* Remove feature flag from `wp-config.php`. Make sure question category functionality works.
